examples/keras_recipes/approximating_non_function_mappings.py

```diff
@@ -0,0 +1,424 @@
+# -*- coding: utf-8 -*-
+"""
+Title: Approximating non-Function Mappings with Mixture Density Networks
+Author: [lukewood](https://twitter.com/luke_wood_ml)
+Date created: 2023/07/15
+Last modified: 2023/07/15
+Description: Approximate non one to one mapping using mixture density networks.
+Accelerator: CPU
+"""
+
+"""
+## Approximating NonFunctions
+
+Neural networks are universal function approximators. Key word: function!
+While powerful function approximators, neural networks are not able to
+approximate non-functions.
+One important restriction to remember about functions - they have one input, one
```

Comment:
Perhaps...

One important characteristic of functions is that they map one input to a one unique output.
Neural networks suffer greatly when the training set has multiple values of Y for a single X.